pycbc/workflow/jobsetup.py

```diff
@@ -290,8 +290,12 @@ def multi_ifo_coherent_job_setup(workflow, out_files, curr_exe_job,
         frame_files = datafind_outs[:-2]
     else:
         ipn_sky_points = None
-        bank_veto = datafind_outs[-1]
-        frame_files = datafind_outs[:-1]
+        if 'bank_veto_bank' in datafind_outs[-1].name:
+            bank_veto = datafind_outs[-1]
+            frame_files = datafind_outs[:-1]
+        else:
+            bank_veto = None
+            frame_files = datafind_outs
```

Comment:
This function seems particularly fragile (and the comment just above the change appears to agree). Looping over `datafind_outs` until you find one with the right name seems a better way, would that not work?

Reply:
This code is definitely questionable. I can look into rewriting it.